platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/HappyPathTest.java

```diff
@@ -10,20 +10,21 @@
 import org.hiero.otter.fixtures.TimeManager;
 import org.hiero.otter.fixtures.Validator.LogFilter;
 import org.hiero.otter.fixtures.Validator.Profile;
+import org.hiero.otter.fixtures.turtle.TurtleNetwork;
 import org.junit.jupiter.api.Disabled;
 
 public class HappyPathTest {
 
     @Disabled
     @OtterTest
     void testHappyPath(TestEnvironment env) throws InterruptedException {
+        env.start();
```

Comment:
This should not be necessary. If the tests start, the environment should be ready to go.

Reply:
Moved as part of the TestEnvironment lifecycle.